.github/workflows/test.yaml

```diff
@@ -55,13 +55,15 @@ jobs:
       script: "ci/test_python_dask.sh"
   conda-python-scikit-learn-accel-tests:
     secrets: inherit
-    uses: rapidsai/shared-workflows/.github/workflows/conda-python-tests.yaml@branch-25.04
+    uses: rapidsai/shared-workflows/.github/workflows/conda-python-tests.yaml@branch-25.06
     with:
       build_type: ${{ inputs.build_type }}
       branch: ${{ inputs.branch }}
       date: ${{ inputs.date }}
       sha: ${{ inputs.sha }}
       script: "ci/test_python_scikit_learn_tests.sh"
+      # Select amd64 architecture and latest Python version, running one job per major CUDA version
+      matrix_filter: ap(select(.ARCH == "amd64")) | group_by(.CUDA_VER|split(".")|map(tonumber)|.[0]) | map(max_by([.CUDA_VER,.PY_VER]|map(split(".")|map(tonumber))))'
```

Comment:
```suggestion
      matrix_filter: map(select(.ARCH == "amd64")) | group_by(.CUDA_VER|split(".")|map(tonumber)|.[0]) | map(max_by([.CUDA_VER,.PY_VER]|map(split(".")|map(tonumber))))'
```

You're missing a character here.

Reply:
There was indeed a syntax error in the nightlies configuration, however that error was not present in the pr.yaml configuration and no job was selected regardless. However, I believe I found the issue. One must ensure that the matrix selection yields a list and not just a single configuration.